autopilot/prefs.py

```diff
@@ -345,15 +345,15 @@ class Scopes(Enum):
     },
     'NCHANNELS': {
         'type': 'int',
-        'text': "Number of Audio channels",
+        'text': "Number of Audio channels (deprecated; used OUTCHANNELS)",
```

Comment:
this makes me think that we should probably have some more specific means of marking deprecation. I think using `DeprecationWarning` would be fine for now, and we can add an additional `"deprecation"` field to a pref? Though that seems piecemeal to me, it seems like there won't be a good 'unified' way of doing deprecation warnings since they will occur in many different contexts.

Reply:
done here https://github.com/wehr-lab/autopilot/pull/102/commits/ab37be48ab71b822ba10c8706af3ef77b560b1be